src/main/java/com/ibm/eventstreams/connect/mqsink/JMSWriter.java

```diff
@@ -15,12 +15,20 @@
  */
 package com.ibm.eventstreams.connect.mqsink;
 
+import com.ibm.eventstreams.connect.mqsink.builders.MessageBuilder;
 import com.ibm.mq.MQException;
 import com.ibm.mq.constants.MQConstants;
-import com.ibm.mq.jms.*;
-import com.ibm.eventstreams.connect.mqsink.builders.MessageBuilder;
+import com.ibm.mq.jms.MQConnectionFactory;
+import com.ibm.mq.jms.MQQueue;
 import com.ibm.msg.client.wmq.WMQConstants;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.errors.RetriableException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import javax.jms.*;
```

Comment:
Could instead be:
`import javax.jms.DeliveryMode;
import javax.jms.JMSContext;
import javax.jms.JMSException;
import javax.jms.JMSProducer;
import javax.jms.JMSRuntimeException;
import javax.jms.Message;`

Like it was before.